samples/highcharts/chartchooser/continuous-distribution-scatter/demo.js

```diff
@@ -0,0 +1,110 @@
+Highcharts.setOptions({
+    colors: ['#058DC7', '#50B432', '#ED561B', '#DDDF00', '#24CBE5', '#64E572', '#FF9655', '#FFF263', '#6AF9C4']
+});
+
+const discipline = [{
+    name: "Basketball",
+    data: "basketball"
+},
+{
+    name: "Triathlon",
+    data: "triathlon"
+},
+{
+    name: "Volleyball",
+    data: "volleyball"
+}
+];
+
+
+async function getData() {
+    const response = await fetch("https://cdn.jsdelivr.net/gh/highcharts/highcharts@24912efc85/samples/data/olympic2012.json?callback=?"
```

Comment:
Remove the `?callback=?` part, which is not in use. This part causes the CircleCI tests to fail because the test data is not rewritten properly.

Reply:
Done